Libraries/Tile/Tile.h

```diff
@@ -0,0 +1,195 @@
+#ifndef Tile_h
+#define Tile_h
+
+class Tile{
+  private:
```

Comment:
Any private member please write it with the name ending on _ and in english:
first_data_;
second_data_;
And also, if you want you could also use an array like:
char data_[2];